heartbeat/docs/getting-started.asciidoc

```diff
@@ -1,184 +1,73 @@
-[[heartbeat-getting-started]]
-== Get started with {beatname_uc}
+[id="{beatname_lc}-installation-configuration"]
+== {beatname_uc} quick start: installation and configuration
 
 ++++
-<titleabbrev>Get started</titleabbrev>
+<titleabbrev>Quick start: installation and configuration</titleabbrev>
 ++++
 
-include::{libbeat-dir}/shared-getting-started-intro.asciidoc[]
+This guide describes how to get started quickly collecting uptime data about
+your hosts. You'll learn how to:
 
-* <<heartbeat-installation>>
-* <<heartbeat-configuration>>
-* <<heartbeat-template>>
-* <<load-kibana-dashboards>>
-* <<heartbeat-starting>>
-* <<view-kibana-dashboards>>
-* <<setup-repositories>>
+* install {beatname_uc}
+* specify the protocols to monitor
+* send uptime data to {es}
+* visualize the uptime data in {kib}
 
+// REVIEWERS: This screenshot comes from demo.elastic.co. Let me know if you
+// have a build or screenshot that you'd prefer to use here. 
 
-[[heartbeat-installation]]
+[role="screenshot"]
+image::./images/heartbeat-statistics.png[{beatname_uc} HTTP monitoring dashboard]
```

Comment:
What about showing an uptime app screenshot instead?

Reply:
IMO showing the app screenshot *instead* of the dashboards muddies the waters a bit for pure OSS users. Also, Observability users will still need to use the finer-grained dashboards to see stuff that isn't visible in the apps.

As a follow-on PR, I'd like to make the "What's next" section a bit more graphical. Maybe we can also add a section (flagged clearly as X-Pack) that showcases the app.  It is a little hard to keep those screen captures up-to-date, though, especially since the team that makes the UI changes won't be updating the Beats docs. WDYT?

Reply:
> As a follow-on PR, I'd like to make the "What's next" section a bit more graphical. Maybe we can also add a section (flagged clearly as X-Pack) that showcases the app.

I like this idea.

> It is a little hard to keep those screen captures up-to-date, though, especially since the team that makes the UI changes won't be updating the Beats docs. WDYT?

Fair point. With the addition of a new observability docs repo though, I think it'd be worthwhile to have one screenshot of each app that we can single source and update at least every other release.